js/core/utils/ready_callbacks.js

```diff
@@ -19,8 +14,7 @@ const subscribeReady = callOnce(() => {
 const readyCallbacks = {
     add: callback => {
         const windowExists = hasWindow();
-
-        if(windowExists && isReady()) {
+        if(windowExists && domAdapter.getReadyState() !== 'loading') {
```

Comment:
So what about the 'interactive' value mentioned in the Note?

Reply:
It's not obvious why did you made such changes, If everything is correct it need an explanation in the description.

Reply:
We don't need extra checking when IE returns 'interactive' value too early.
So It need only checking that value is not 'loading'